src/Raven.Client/Documents/Queries/QueryResultBase.cs

```diff
@@ -76,6 +76,12 @@ public abstract class QueryResultBase<TResult, TInclude>
         /// </summary>
         public string NodeTag { get; set; }
 
+        /// <summary>
+        /// If the query returned time series results, this field will contain
+        /// the names of the fields with time series data
+        /// </summary>
+        public List<string> TimeSeriesFields { get; set; }
```

Comment:
This is metadata needed by the Studio. It should not be part of Client API IMO. We probably should move this to QueryResultServerSide?

Reply:
Or even to DocumentQueryResult?